### ISSUE TITLE
Learn and properly document using Docker, create an image that can be replicated instantly after installing Docker on a new machine

### ISSUE THREAD

**Issue by 1unarzDev** [MHSeals/mhsboat_ctrl#31]
TL;DR, Docker is important, we need a container that installs every dependency in a contained environment, caches it for later use, and can be updated after being pushed to GitHub.

When configured correctly and in a way that can be expanded quickly, Docker can streamline the initial development startup process. We wasted 2 months this year (for newer members) getting the development environment set. This isn't necessarily a bad thing since it teaches you about the Linux environment, but it makes the club far less appealing to be in. Moreover, Docker should help us update the project without running into issues caused by incorrect dependencies.
